.circleci/config.yml

```diff
@@ -4,6 +4,7 @@ defaults: &defaults
       environment:
         LANG: C.UTF-8
         EMTEST_DETECT_TEMPFILE_LEAKS: 1
+        EMCC_ERROR_ON_BUILDING_SYSTEM_LIBS: 1 # disallow system lib building in general - only the initial build should do it, for everyone
```

Comment:
Would a better name be `EMCC_NO_ON_DEMAND_SYSTEM_LIB_BUILDS` or something like that? `ERROR_ON_BUILDING_SYSTEM_LIBS` makes this read like `ERROR_ON_UNDEFINED_SYMBOLS` or `ERROR_ON_MISSING_LIBRARIES`, i.e. as if it was an error if system libs build fails (but otherwise might just have been a warning if system lib build fails?)

Reply:
Or alternatively `EMCC_BUILD_SYSTEM_LIBS_ON_DEMAND=0` to make the option read positive instead of negative? (undefined meaning ==1)

Reply:
I like `EMCC_BUILD_SYSTEM_LIBS_ON_DEMAND`. Would also make it easier to make 0 be the default in the future if we want to pre-distribute all the libs in the SDK.

Reply:
Sounds good, renaming.